javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+var contactForm = document.querySelector('form');
+console.log(contactForm);
+contactForm.addEventListener('submit', contactFunction);
+
+function contactFunction(event) {
+  event.preventDefault();
+  var newObject = {};
+  newObject.name = contactForm.elements.name.value;
+  newObject.email = contactForm.elements.email.value;
+  newObject.message = contactForm.elements.message.value;
+  console.log('Message Data:', newObject);
```

Comment:
The only thing to add here is to make sure that you reset the form after it is submitted. Check out the `reset` method on MDN.